corehq/apps/export/static/export/js/models.js

```diff
@@ -210,6 +232,12 @@ Exports.ViewModels.ExportItem.prototype.isCaseName = function() {
     return this.path()[this.path().length - 1].name === 'name';
 };
 
+Exports.ViewModels.ExportItem.prototype.readablePath = function() {
+    return _.map(this.path(), function(pathNode) {
+        return pathNode.name();
+    }).join('.')
```

Comment:
OMG

On Thu, Apr 14, 2016 at 10:37 AM, dimagimon <notifications@github.com>
wrote:

> In corehq/apps/export/static/export/js/models.js
> <https://github.com/dimagi/commcare-hq/pull/11266#discussion_r59728327>:
>
> > @@ -210,6 +232,12 @@ Exports.ViewModels.ExportItem.prototype.isCaseName = function() {
> >      return this.path()[this.path().length - 1].name === 'name';
> >  };
> >
> > +Exports.ViewModels.ExportItem.prototype.readablePath = function() {
> > +    return _.map(this.path(), function(pathNode) {
> > +        return pathNode.name();
> > +    }).join('.')
>
> Missing semicolon. (semi)
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/11266/files/5c4d5f2ddda487b6e928affb475d20d44beeea9f#r59728327>
>